spring-cloud-dataflow-admin-kubernetes/src/main/java/org/springframework/cloud/dataflow/module/deployer/kubernetes/DefaultContainerFactory.java

```diff
@@ -38,6 +39,9 @@
 	@Autowired
 	protected KubernetesModuleDeployerProperties properties;
 
+	@Autowired
+	private AdminProperties adminProperties;
+
```

Comment:
Of note, the same 'contextLoads' test fails for admin-cloudfoundry.  Investigating.